armi/physics/neutronics/tests/test_crossSectionManager.py

```diff
@@ -93,11 +94,69 @@ def setUp(self):
         self.bc.extend(self.blockList)
 
     def test_createRepresentativeBlock(self):
-
         avgB = self.bc.createRepresentativeBlock()
         self.assertAlmostEqual(avgB.p.percentBu, 50.0)
 
 
+class TestCylindricalComponentsAverageBlockCollection(unittest.TestCase):
```

Comment:
`CylindricalComponentsAverageBlockCollection::_makeRepresentativeBlock` has coverage already in `TestBlockCollectionComponentAverage1DCylinder::test_ComponentAverage1DCylinder` so this test may not be needed. It's just buried in a more "in-repo integration test" style. 

Thoughts?

Reply:
sure. Done.